skyportal/handlers/api/candidate.py

```diff
@@ -412,10 +423,10 @@ def get(self, obj_id=None):
             "null",
             "undefined",
         ]:
-            start_date = arrow.get(start_date).datetime
+            start_date = arrow.get(start_date).isoformat()
             q = q.filter(Candidate.passed_at >= start_date)
         if end_date is not None and end_date.strip() not in ["", "null", "undefined"]:
-            end_date = arrow.get(end_date).datetime
+            end_date = arrow.get(end_date).isoformat()
```

Comment:
This implementation is more stable for me locally, using datetimes runs into some issues with UTC on my mac OSX.